### PR TITLE
修改了 Tensors of scalars

### PR DESCRIPTION
**Pull request by YiYiXia**
将原先的：下列代码将创建由 int32 组成的4x3密集张量：
改成了：下列代码将创建由 float32 组成的4x3密集张量：

### CODE REVIEW

File: locales/zh_CN/LC_MESSAGES/scalar_tensor.po [hunk 1]
```diff
@@ -50,16 +50,15 @@ msgstr ""
...
 #: ../../scalar_tensor.rst:23
 msgid "x = ti.var(ti.f32, shape=(4, 3))"
 msgstr ""
 
 #: ../../scalar_tensor.rst:25
 msgid ""
-"If shape is ``()`` (empty tuple), then a 0-D tensor (scalar) is created: "
-"::"
+"If shape is ``()`` (empty tuple), then a 0-D tensor (scalar) is created: ::"
 msgstr "如果形状设为 ``()`` （空tuple），那将创建一个0维张量（即标量）： ::"
```
Comment (isdanni) [inline locales/zh_CN/LC_MESSAGES/scalar_tensor.po]:
```suggestion
msgstr "如果形状设为 ``()`` （空元组），那将创建一个0维张量（即标量）： ::"
```
We could stick to the Chinese version of tuple here ;-))

File: locales/zh_CN/LC_MESSAGES/scalar_tensor.po [hunk 2]
```diff
@@ -93,14 +94,15 @@ msgstr ""
 #: ../../scalar_tensor.rst:44
 #, fuzzy
 msgid ""
-"Not providing ``shape`` allows you to *place* the tensor in a layout "
-"other than the default *dense*, see :ref:`layout` for more details."
-msgstr "不提供形状 ``shape`` 还可以使你定义张量为 *稀疏* 张量，见 :ref:`sparse` 。"
+"Not providing ``shape`` allows you to *place* the tensor in a layout other than "
+"the default *dense*, see :ref:`layout` for more details."
+msgstr ""
+"不提供形状 ``shape`` 还可以使你定义张量为 *稀疏* 张量，见 :ref:`sparse` 。"
```
Comment (isdanni) [inline locales/zh_CN/LC_MESSAGES/scalar_tensor.po]: needs help here :hear_no_evil: Someone please corrects me if I'm wrong. Is the link for `sparse` or `layout`? 
I checked the latest version it seems like the sparse is still WIP?
```suggestion
"不提供形状 ``shape`` 还可以使你定义张量为 *稀疏* 张量，参考见 :ref:`layout` 。"
```

**Review by isdanni — COMMENTED** [review]
Thank you for your contribution! Added a few suggestions  :+1:

File: locales/zh_CN/LC_MESSAGES/scalar_tensor.po (hunk 2 above)
Comment (YiYiXia) [reply locales/zh_CN/LC_MESSAGES/scalar_tensor.po]: wow,It's my first time to use "pull request".I don't know whether I did it right.
Should I modify the part you mentioned above and then merge again?

Comment (isdanni) [reply locales/zh_CN/LC_MESSAGES/scalar_tensor.po]: > wow,It's my first time to use "pull request".I don't know whether I did it right.
> Should I modify the part you mentioned above and then merge again?

You could directly apply suggested changes here; After that, it should show suggestion resolved; After others review your pull request, we can merge it to master :-)

File: locales/zh_CN/LC_MESSAGES/scalar_tensor.po [hunk 3]
```diff
@@ -191,14 +192,14 @@ msgstr ""
 
 #: ../../scalar_tensor.rst:99
 msgid ""
-"In Python, x[(exp1, exp2, ..., expN)] is equivalent to x[exp1, exp2, ...,"
-" expN]; the latter is just syntactic sugar for the former."
+"In Python, x[(exp1, exp2, ..., expN)] is equivalent to x[exp1, exp2, ..., "
+"expN]; the latter is just syntactic sugar for the former."
 msgstr ""
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/scalar_tensor.po:197]: Not trannslated?

**Review by archibate — COMMENTED** [review]
Thank for adding this and sorry for my delayed reply! LGTM in general, but note that some entries are not translated yet, is this intentional?

File: locales/zh_CN/LC_MESSAGES/scalar_tensor.po (hunk 3 above)
Comment (YiYiXia) [reply locales/zh_CN/LC_MESSAGES/scalar_tensor.po:197]: Oh,I just find a small mistake and want to change it,so I don't know why translator didn't translate these words before.(´･㉨･`)

Comment (archibate) [reply locales/zh_CN/LC_MESSAGES/scalar_tensor.po:197]: Oh I see, added #82 for this, thank for the contribution, merging now :)

**Pull request merged**